RiotSwiftUI/Modules/Room/PollHistory/Coordinator/PollHistoryCoordinator.swift

```diff
@@ -44,11 +44,26 @@ final class PollHistoryCoordinator: Coordinator, Presentable {
     func start() {
         MXLog.debug("[PollHistoryCoordinator] did start.")
         pollHistoryViewModel.completion = { [weak self] result in
-            self?.completion?()
+            switch result {
+            case .genericError:
+                self?.showErrorAlert()
+            }
         }
     }
     
     func toPresentable() -> UIViewController {
         pollHistoryHostingController
     }
 }
+
+private extension PollHistoryCoordinator {
+    func showErrorAlert() {
+        let alert = UIAlertController(title: VectorL10n.pollHistoryFetchingError,
+                                      message: nil,
+                                      preferredStyle: .alert)
+        
+        let cancelAction = UIAlertAction(title: VectorL10n.ok, style: .cancel)
+        alert.addAction(cancelAction)
+        pollHistoryHostingController.present(alert, animated: true, completion: nil)
```

Comment:
The view model can present these errors itself through the use of `AlertInfo` - there should be plenty of examples of this elsewhere in the codebase :)

Reply:
Yeah, I forgot about it!
Applied here -> https://github.com/vector-im/element-ios/pull/7303/commits/6caf8d707d7e9f516cadbf1f638b980965a5f25c